### ISSUE TITLE
User should be able to click on "SecretSwap" and go back to secretswap.io

### ISSUE THREAD

**Issue by lauracw100** [scrtlabs/EthereumBridgeFrontend#147]
The logo isn't clickable, but I'm thinking if the user comes to SecretSwap from secretswap.io and the "Use SecretSwap" button, they should be able to go back to that entry point.

Suggestion: add a link to the logo/SecretSwap verbiage to go to https:://secretswap.io

**Comment by cankisagun** [comment]
Let's do it super easy fix

**Comment by alejandrordzl** [comment]
Yeah, super easy fix, already deployed here [https://swap-app.onrender.com/swap](url)
